### DISCUSSION TITLE
Custom Tracks

### DISCUSSION THREAD

**Discussion by duhhbzz** [stepmania/stepmania#2170] [Q&A]
I'm sure there's a forum thread or possibly even documentation on it somewhere, but can I be directed to or told if there is a general template or default for the .sm files for the tracks. I understand there is a process in-game by editing then the game will create a .sm file for you, however if I was to use an editor such is VScode or notepad even to write my own information, what is the general template / attributes to include or required?

**Comment by Anders429** [comment]
The wiki has a page about the SSC format that lists all of the tags: https://github.com/stepmania/stepmania/wiki/ssc
